pkgs/by-name/e1/e1s/package.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "e1s";
+  version = "1.0.34";
```

```suggestion
let
  pname = "e1s";
  version = "1.0.34";
in
buildGoModule {
```
Using `rec` is discouraged, you can learn more here: https://nix.dev/guides/best-practices#recursive-attribute-set-rec